cisco_umbrella_reporting/plugin.spec.yaml

```
@@ -0,0 +1,121 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: cisco_umbrella_reporting
+title: Cisco Umbrella Reporting
+version: 1.0.0
+description: This plugin utilizes Cisco Umbrella to get the most complete view of the relationships
+  and evolution of Internet domains, IP addresses, and autonomous systems to pinpoint
+  attackers infrastructures and predict future threats
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/cisco_umbrella_reporting
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.cisco.com
+tags:
+- cisco
+- umbrella
+- reporting
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [cisco, umbrella, reporting]
+  features: []
+connection:
+  api_key:
+    title: API Key
+    description: Enter API key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  api_secret:
+    title: API Secret Key
+    description: Enter secret key
+    type: credential_secret_key
+    required: true
+    example: 9de5069c5afe602b2ea0a04b66beb2c0
+  organization_id:
+    title: Organization ID
+    description: ID of your organization
```

```suggestion
    description: ID of your Cisco Umbrella organization
```
Specify Cisco Umbrella or else it may cause confusion to users :)